storage/innobase/xtrabackup/test/t/ib_external_xa_prepared.sh

```diff
@@ -0,0 +1,42 @@
+############################################################################
+# Test external XA prepare transactions in backup result are not rolled back
+# after --prepare
+############################################################################
+
+. inc/common.sh
+
+vlog "start server with gtid enabled"
+start_server --gtid-mode=ON --enforce-gtid-consistency=ON --log-bin --log-slave-updates
+
+run_cmd $MYSQL $MYSQL_ARGS test -e "create table t1(id int, primary key(id)) engine=innodb"
```

Comment:
can you please use `mysql` instead of `run_cmd $MYSQL $MYSQL_ARGS`

Reply:
Sure.